recipe/meta.yaml

```diff
@@ -112,6 +108,44 @@ app:
   summary: The Scientific Python Development Environment
   type: desk
 
+outputs:
+  - name: spyder-base
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
```

Comment:
i'm about 98% sure you can remove these two lines and revert your renamed files.

Reply:
hmm... I don't want `build.sh` or `bld.bat` to run for `spyder`, only for `spyder-base`. I thought these were run for both packages if found in the recipe directory, so required explicit reference.

Reply:
Am I mistaken about that?

Reply:
i think it will be fine...

Reply:
I've reverted the file rename and can confirm that `build.sh` and `bld.bat` only run for the parent package, `spyder-base` in this case.

Thanks for the suggestion.